react/features/base/dialog/components/AbstractDialog.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from 'react';
+
+import { hideDialog } from '../actions';
+
+/**
+ * Abstract dialog to display dialogs.
+ */
+export default class AbstractDialog extends Component {
+
+    /**
+     * Abstract Dialog component's property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * Whether cancel button is disabled. Enabled by default.
+         */
+        cancelDisabled: React.PropTypes.bool,
+
+        /**
+         * Optional i18n key to change the cancel button title.
+         */
+        cancelTitleKey: React.PropTypes.string,
+
+        /**
+         * Used to show hide the dialog on cancel.
+         */
+        dispatch: React.PropTypes.func,
+
+        /**
+         * The handler for onCancel event.
+         */
+        onCancel: React.PropTypes.func,
+
+        /**
+         * The handler for the event when submitting the dialog.
+         */
+        onSubmit: React.PropTypes.func,
+
+        /**
+         * Whether to show/hide the dialog.
+         */
+        show: React.PropTypes.bool,
+
+        /**
+         * Is submit button enabled/disabled. Enabled by default.
+         */
+        submitDisabled: React.PropTypes.bool,
```

Comment:
I was adding it as we had a disable for the other one ... Maybe having only cancel button, this were my thoughts.